keyboards/ergodox_infinity/rules.mk

```diff
@@ -37,7 +36,6 @@ LCD_WIDTH = 128
 LCD_HEIGHT = 32
 
 # project specific files
-SRC = matrix.c \
-      led.c
+SRC = led.c
```

Comment:
I think `led.c` can be removed altogether, it doesn't look like it does anything.

Reply:
True, it's just a placeholder. Removing it.